core-bundle/src/Doctrine/Backup/BackupManager.php

```diff
@@ -27,22 +27,32 @@ class BackupManager
 
     private Connection $connection;
     private DumperInterface $dumper;
-    private string $backupDir;
+    private VirtualFilesystemInterface $vfs;
     private array $tablesToIgnore;
     private RetentionPolicyInterface $retentionPolicy;
 
-    public function __construct(Connection $connection, DumperInterface $dumper, string $backupDir, array $tablesToIgnore, RetentionPolicyInterface $retentionPolicy)
+    public function __construct(Connection $connection, DumperInterface $dumper, VirtualFilesystemInterface $vfs, array $tablesToIgnore, RetentionPolicyInterface $retentionPolicy)
```

Comment:
What do you think about naming the parameter `VirtualFilesystemInterface $backupsStorage` to be in line with how you would use it with autowiring?

Reply:
Agreed, done :)